distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Ref.enso

```diff
@@ -52,6 +52,6 @@ type Ref
           Increment the integer stored in a ref.
 
           (Ref.new 10) . modify (_+1) . get
-          # => 11
+          # => 10
```

Comment:
I'm pretty sure that this example is still wrong. Are you sure you have tested it?

Please add a test that tests exactly this example, copied as close to the original code as possible to ensure it is correct (in the future we definitely want to have #7949, but until that happens, at least the less trivial examples should just be replicated into the test suite).

Reply:
The problem with this example is that as far as I understand the code, `modify` returns the _previous value_ of the `Ref`.

So `(Ref.new 10) . modify (_+1)` will return `10`. But then you call `get` on it - so I assume this will fail with 'unknown method get of Integer'.

Reply:
Added example test.